odk1-src/form-functions.rst

```diff
@@ -0,0 +1,719 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Colores
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  arctan
+  arg
+  bday
+  bp
+  br
+  chkn
+  colores
+  fsh
+  gustan
+  jr
+  nodeset
+  prefs
+  seleccionados
+  te
+  tres
+  π
+  
+  
+	
+******************************
+Form Functions and Operators
```

Comment:
Let's split into two documents.

Reply:
I'm not so sure about this one! Operators can join the results of functions. I think of this document as being about the building blocks of expressions.

Reply:
If y'all don't want to split, that's fine by me. But at the minimum maybe it should be called `Form Operators and Functions` to match the order?

Reply:
I agree with this last point. I'll change the file name too.